src/database/seeders/ProjectSettingTypesTableSeeder.php

```diff
@@ -62,6 +62,11 @@ public function run()
                 ]),
                 'is_translatable' => false,
             ],
+            [
+                'name' => 'email',
+                'validation_rules' => 'sometimes|string|min:2|max:40000',
```

Comment:
the min and max are not logical.

please don't depend on handling multiple emails in the same key unless you will have a specific separator and validate everyone if it's an email. otherwise you may stick to just one email and make sure to validate it as email with suitable min and max values